src/main/java/com/mparticle/kits/PilgrimSDKKit.java

```diff
@@ -0,0 +1,183 @@
+package com.mparticle.kits;
+
+import android.content.Context;
+
+import com.foursquare.pilgrim.LogLevel;
+import com.foursquare.pilgrim.PilgrimSdk;
+import com.foursquare.pilgrim.PilgrimUserInfo;
+import com.mparticle.MParticle;
+import com.mparticle.consent.ConsentState;
+import com.mparticle.identity.MParticleUser;
+
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * This is the PilgrimSDK mParticle kit , used to extend the functionality of mParticle SDK and allow control of
+ * the PilgimSDK; Mapping analogous public mParticle APIs into PilgrimSDK's API.
+ * <p>
+ * <p>
+ * In addition to this file, you also will need to edit:
+ * - ./build.gradle (as explained above)
+ * - ./README.md
+ */
+final public class PilgrimSDKKit extends KitIntegration implements KitIntegration.UserAttributeListener, KitIntegration.IdentityListener {
```

Comment:
Can we call this `PilgrimSdkKit` to lineup with our main class which is `PilgrimSdk`?

Reply:
Sure